lndrpc.proto

```diff
@@ -28,13 +28,39 @@ package lnrpc;
 // The WalletUnlocker service is used to set up a wallet password for
```

Comment:
I think we should be tracking the upstream versions of Lightning and Raiden networks we are developing against. Maybe Include the version of protobufs for now?

Reply:
Yeah, for now I've just been trying to keep up to date with the latest since all projects involved are in rapid development. I was just of listing the targeted versions of Raiden & lnd in our readme/documentation and docker, do you have a better idea? I don't think Raiden has protobufs.

Reply:
Although it's a bit weird, maybe we should define these versions as constants in our project, and also log it when the lnd\raiden client is connecting. That way, in case of a compatibility problem, the user will know immediately what was the expected version.